build.gradle

```diff
@@ -11,12 +12,46 @@ repositories {
 }
 
 dependencies {
-    compile "org.gradle:gradle-tooling-api:4.0"
+    compile "org.gradle:gradle-tooling-api:4.5"
     compile group: 'commons-cli', name: 'commons-cli', version: '1.4'
-    compile 'com.google.code.gson:gson:2.8.0'
+    compile 'com.google.code.gson:gson:2.8.2'
+    compile 'me.tongfei:progressbar:0.5.5'
     runtime 'org.slf4j:slf4j-simple:1.7.10'
 }
 
+task generateVersion {
```

Comment:
Just add some comment here explaining what this task do an whats for.

Reply:
Done in next commit